protocol/x/stats/keeper/keeper_test.go

```diff
@@ -290,3 +293,19 @@ func TestExpireOldStats(t *testing.T) {
 	k.ExpireOldStats(ctx)
 	require.NotNil(t, k.GetEpochStatsOrNil(ctx, uint32(12)))
 }
+
+func TestGetStakedAmount(t *testing.T) {
+	tApp := testapp.NewTestAppBuilder(t).Build()
+	ctx := tApp.InitChain()
+	statsKeeper := tApp.App.StatsKeeper
+	stakingKeeper := tApp.App.StakingKeeper
+	expectedCoinsStaked := big.NewInt(100)
```

Comment:
Add multiple test cases with different values. Test realistic values like 1e20 (100 whole coins), 1e23 (100k whole coins)

Reply:
Also, add test case for cache invalidation: previous cache was set >15 minutes ago; retur new value != cached value.

Reply:
Did we address this?

Reply:
Missed this! addressed now